src/commands/codepush/patch.ts

```diff
@@ -0,0 +1,101 @@
+import { AppCommand, CommandArgs, CommandResult, help, failure, ErrorCodes, success, getCurrentApp, shortName, longName, required, hasArg, position, name } from "../../util/commandline";
+import { out } from "../../util/interaction";
+import { inspect } from "util";
+import { MobileCenterClient, models, clientRequest, clientCall } from "../../util/apis";
+import * as chalk from "chalk";
+import * as semver from "semver";
+
+const debug = require("debug")("mobile-center-cli:commands:codepush:patch");
+
+@help("Update the metadata for an existing release")
+export default class PatchCommand extends AppCommand {
+  
+  @help("CodePush deployment name.")
+  @required
+  @name("ExistingDeploymentName")
```

Comment:
Typical CLI conventions use all lower cases and dashes rather than PascalCase. So `--existing-deployment-name` not `--ExistingDeploymentName`

Reply:
fixed.